.travis.yml

```diff
@@ -9,7 +9,9 @@ services:
 - docker
 before_install:
 - pip install flake8
-- npm install -g eslint stylelint
+- nvm install 12.18.3
```

Comment:
We can also use version `12.8.1` which comes pre-installed.

Reply:
```
You need to run "nvm install 12.18.1" to install it before using it.

The command "nvm use 12.18.1" failed and exited with 3 during .
```